packaging/opensuse/snapd.spec

```diff
@@ -352,6 +352,8 @@ fi
 %{_datadir}/dbus-1/services/io.snapcraft.Launcher.service
 %{_datadir}/dbus-1/services/io.snapcraft.Settings.service
 %{_sysconfdir}/xdg/autostart/snap-userd-autostart.desktop
+/lib/systemd/system-generators/snapd-generator
```

Comment:
This should be `%{_systemdgeneratordir}/snapd-generator`

Also, shouldn't this also be in the Fedora spec?